sceneview/src/main/java/io/github/sceneview/utils/FrameTime.kt

```diff
@@ -26,13 +25,13 @@ data class FrameTime(val nanoseconds: Long, val lastNanoseconds: Long? = null) {
      * ### The duration between this frame and the last frame
      */
     fun interval(lastNanoseconds: Long?): Duration =
-        abs((lastNanoseconds ?: 0) - nanoseconds).nanoseconds
+        (nanoseconds - (lastNanoseconds ?: 0)).nanoseconds
```

Comment:
Could you please check where it is called because I think I sometimes inverted parameters from previous to actual frame to reduce null checks. Maybe in `PlaneRenderer`, `ArModelNode`,...

Reply:
I've found the following usages, however, the nullable `ArFrame` is first used and when updated:
- https://github.com/SceneView/sceneview-android/blob/bbe8cc4d93f11b94db9b6d57543e4ce4901e54a8/arsceneview/src/main/java/io/github/sceneview/ar/node/ArModelNode.kt#L258
- https://github.com/SceneView/sceneview-android/blob/9d31efcbaaff918069ee084b1335fe9cb9809f13/arsceneview/src/main/java/io/github/sceneview/ar/scene/PlaneRenderer.kt#L129

I think we can refactor the code to avoid using nullable values in the `FrameTime`.